site/da_DK.all.json

```diff
@@ -1126,18 +1126,18 @@
     "other": "Komplet"
   },
   "shopping_complete_subscription": {
-    "other": "Dit abonnementskøb blev gennemført. Du abonnerer nu på {{ .Title }} ."
+    "other": "Dit abonnementskøb blev gennemført. Du abonnerer nu på {{.Title}} ."
   },
   "shopping_complete_subscription_browse": {
     "other": "Du er velkommen til at <a href=\"/\">browse rundt</a> på siden, eller <a href=\"/search.html\">søg</a> efter noget specifikt."
   },
   "shopping_info_plan_offer": {
-    "one": "{{ .Interval }} automatisk hvert .Interval",
-    "other": "{{ .Count }} automatisk hver {{ .Interval }}"
+    "one": "{{.Interval}} automatisk hvert .Interval",
```

Comment:
this has .Interval in there twice, with the second not in moustaches. not sure if thats intended or not?